README.md

```diff
@@ -1 +1,235 @@
-# terraform-google-bigquery-dataset-iam
\ No newline at end of file
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="400"/>][homepage]
+
+[![Terraform Version][badge-terraform]][releases-terraform]
+[![Google Provider Version][badge-tf-gcp]][releases-google-provider]
+[![Join Slack][badge-slack]][slack]
+
+# terraform-terraform-google-bigquery-dataset-iam
+
+A [Terraform] module for [Google Bigquery IAM][google-bigquery-iam] on [Google Cloud Platform (GCP)][gcp].
+
+**_This module supports Terraform version 1
+and is compatible with the Terraform Google Provider version 3._**
+
+This module is part of our Infrastructure as Code (IaC) framework
+that enables our users and customers to easily deploy and manage reusable,
+secure, and production-grade cloud infrastructure.
+
+- [Module Features](#module-features)
+- [Getting Started](#getting-started)
+- [Module Argument Reference](#module-argument-reference)
+  - [Top-level Arguments](#top-level-arguments)
+    - [Module Configuration](#module-configuration)
+    - [Main Resource Configuration](#main-resource-configuration)
+    - [Extended Resource Configuration](#extended-resource-configuration)
+- [Module Attributes Reference](#module-attributes-reference)
+- [External Documentation](#external-documentation)
+- [Module Versioning](#module-versioning)
+  - [Backwards compatibility in `0.0.z` and `0.y.z` version](#backwards-compatibility-in-00z-and-0yz-version)
+- [About Mineiros](#about-mineiros)
+- [Reporting Issues](#reporting-issues)
+- [Contributing](#contributing)
+- [Makefile Targets](#makefile-targets)
+- [License](#license)
+
+## Module Features
+
+A [Terraform] base module for creating a `terraform_google_bigquery_iam_*` resource. This module creates a google bigquery IAM with additional feature to either IAM binding or member and binds into a role depending on `authoritative` variable.
+
+<!--
+These are some of our custom features:
+
+- **Default Security Settings**:
+  secure by default by setting security to `true`, additional security can be added by setting some feature to `enabled`
+
+- **Standard Module Features**:
+  Cool Feature of the main resource, tags
+
+- **Extended Module Features**:
+  Awesome Extended Feature of an additional related resource,
+  and another Cool Feature
+
+- **Additional Features**:
+  a Cool Feature that is not actually a resource but a cool set up from us
+
+- _Features not yet implemented_:
+  Standard Features missing,
+  Extended Features planned,
+  Additional Features planned
+-->
+
+## Getting Started
+
+Most basic usage just setting required arguments:
+
+```hcl
+module "terraform-google-bigquery-dataset-iam" {
+  source = "github.com/mineiros-io/terraform-google-bigquery-dataset-iam.git?ref=v0.1.0"
+}
+```
```

Comment:
required variable are missing in the example as the text states those need to be set.

Reply:
> Most basic usage just setting required arguments:

As the section states: this should be a working minimal example how to call the module.